cms/envs/common.py

```diff
@@ -904,6 +904,7 @@
 
 SOUTH_MIGRATION_MODULES.update({
     'edx_notifications': 'edx_notifications.stores.sql.migrations',
+    'problem_builder': 'problem_builder.south_migrations',
 })
```

Comment:
@antoviaque it would work with edx-platform, but there's a risk that it won't work with workbench (and run tests). I'll check it out.